agent/consul/leader_test.go

```diff
@@ -1283,24 +1286,39 @@ func TestLeader_ConfigEntryBootstrap_Fail(t *testing.T) {
 		}
 	}()
 
-	dir1, s1 := testServerWithConfig(t, func(c *Config) {
-		c.LogOutput = io.MultiWriter(pw, testutil.NewLogBuffer(t))
-		c.Build = "1.6.0"
-		c.ConfigEntryBootstrap = []structs.ConfigEntry{
-			&structs.ServiceSplitterConfigEntry{
-				Kind: structs.ServiceSplitter,
-				Name: "web",
-				Splits: []structs.ServiceSplit{
-					{Weight: 100, Service: "web"},
-				},
+	dir, config := testServerConfig(t)
+	defer os.RemoveAll(dir)
+	config.Build = "1.6.0"
+	config.ConfigEntryBootstrap = []structs.ConfigEntry{
+		&structs.ServiceSplitterConfigEntry{
+			Kind: structs.ServiceSplitter,
+			Name: "web",
+			Splits: []structs.ServiceSplit{
+				{Weight: 100, Service: "web"},
 			},
-		}
-	})
-	defer os.RemoveAll(dir1)
-	defer s1.Shutdown()
+		},
+	}
 
-	result := <-ch
-	require.Empty(t, result)
+	logger := hclog.NewInterceptLogger(&hclog.LoggerOptions{
+		Name:   config.NodeName,
+		Level:  hclog.Debug,
+		Output: io.MultiWriter(pw, testutil.NewLogBuffer(t)),
```

Comment:
This was the only test that wrapped the writer, so instead of adding to the test helper I copied the relevant bits out into the test.